data/base-sigils.yaml

```diff
@@ -0,0 +1,5503 @@
+    - 385
+    - 386
+    - 388
+    - 389
+    - 394
+    - 395
+    - 396
+    - 402
+    - 405
+    - 407
+    - 409
+    - 410
+    - 412
+    - 416
+    - 417
+    - 418
+    - 421
+    - 426
+    - 427
+    - 430
+    - 431
+  induction:
+    spring:
+    - 354
+    - 355
+    - 356
+    - 359
+    - 362
+    - 364
+    - 365
+    - 374
+    - 380
+    - 383
+    - 387
+    - 390
+    summer: 
+    winter:
+    - 363
+    - 370
+    - 371
+    - 373
+    - 375
+    - 381
+    - 382
+    - 384
+    - 391
+    - 400
+    - 403
+    - 404
+    - 414
+    - 415
+    - 420
+    - 423
+    - 425
+    - 428
+  permutation:
+    spring:
+    - 352
+    - 353
+    - 357
+    - 358
+    - 360
+    - 361
+    - 366
+    - 367
+    - 368
+    - 369
+    - 372
+    - 377
+    - 379
+    - 385
+    - 386
+    - 389
+    summer: 
+    winter: 
+  rarefaction:
+    spring: 
+    summer: 
+    winter:
+    - 354
+    - 355
+    - 356
+    - 359
+    - 362
+    - 364
+    - 365
+    - 374
+    - 376
+    - 380
+    - 383
+    - 387
+    - 390
+    - 392
+    - 393
+    - 397
+    - 398
+    - 399
+    - 401
+    - 411
+    - 413
+    - 419
+    - 424
+    - 429
+    - 432
+  antipode:
+    spring:
+    - 352
+    - 358
+    - 366
+    - 369
+    - 372
+    - 389
+    summer: 
+    winter:
+    - 357
+    - 360
+    - 361
+    - 363
+    - 367
+    - 368
+    - 370
+    - 371
+    - 373
+    - 375
+    - 381
+    - 382
+    - 384
+    - 385
+    - 391
+    - 394
+    - 395
+    - 400
+    - 403
+    - 404
+    - 407
+    - 414
+    - 415
+    - 420
+    - 423
+    - 425
+    - 426
+    - 428
+    - 430
+    - 431
+  ascension:
+    spring: 
+    summer: 
+    winter: 
+  clarification:
+    spring: 
+    summer: 
+    winter:
+    - 353
+    - 354
+    - 359
+    - 362
+    - 364
+    - 365
+    - 374
+    - 377
+    - 379
+    - 386
+    - 387
+    - 390
+    - 399
+    - 411
+    - 412
+    - 413
+    - 417
+    - 419
+    - 424
+    - 432
+  decay:
+    spring:
+    - 357
+    - 360
+    - 361
+    - 363
+    - 367
+    - 368
+    - 370
+    - 371
+    - 373
+    - 381
+    - 382
+    - 384
+    - 385
+    - 391
+    summer: 
+    winter:
+    - 352
+    - 358
+    - 366
+    - 369
+    - 372
+    - 378
+    - 388
+    - 389
+    - 396
+    - 402
+    - 405
+    - 409
+    - 410
+    - 416
+    - 418
+    - 421
+    - 427
+  evolution:
+    spring: 
+    summer: 
+    winter: 
+  integration:
+    spring: 
+    summer: 
+    winter:
+    - 355
+    - 356
+    - 376
+    - 380
+    - 383
+    - 392
+    - 393
+    - 397
+    - 398
+    - 401
+    - 429
+  metamorphosis:
+    spring:
+    - 355
+    - 356
+    - 380
+    - 383
+    summer: 
+    winter:
+    - 408
+  nurture:
+    spring: 
+    summer: 
+    winter: 
+  paradox:
+    spring:
+    - 353
+    - 354
+    - 359
+    - 362
+    - 364
+    - 365
+    - 374
+    - 377
+    - 379
+    - 386
+    - 387
+    - 390
+    summer: 
+    winter: 
+  unity:
+    spring: 
+    summer: 
+    winter: 
+other:
```

Comment:
What is `other` section for? There doesn't seem to be any rooms listed. Is this necessary in the file?

Reply:
I can probably get rid of it. The idea was to have a bucket to put rooms in if the script was run in a weird place that wasn't detected as one of the main three cities/regions I am currently supporting.

The better solution might be, for now, to only support Crossing, Shard, and Riverhaven, and only when the associated map image file references either the city name or province name.